modules/swffg-main.js

```diff
@@ -1038,3 +1041,43 @@ Hooks.on("pauseGame", () => {
     }
   }
 });
+
+async function registerCrewRoles() {
+  const defaultArrayCrewRoles = [
+    {
+      "role_name":  game.i18n.localize("SWFFG.Crew.Roles.None"),
+      "role_skill": undefined,
+      "use_weapons": false,
+      "use_handling": false
+    },
+    {
+      "role_name":  game.i18n.localize("SWFFG.Crew.Roles.Pilot_Space"),
+      "role_skill":  game.i18n.localize("SWFFG.SkillsNamePilotingSpace").replace(' ', ' '),
+      "use_weapons": false,
+      "use_handling": true
+    },
+    {
+      "role_name":  game.i18n.localize("SWFFG.Crew.Roles.Gunner.Name"),
+      "role_skill":  game.i18n.localize("SWFFG.SkillsNameGunnery"),
+      "use_weapons": true,
+      "use_handling": false
+    }
+  ];
+  game.settings.registerMenu("starwarsffg", "arrayCrewRoles", {
+    name: game.i18n.localize("SWFFG.Crew.Settings.Name"),
+    label: game.i18n.localize("SWFFG.Crew.Settings.Label"),
+    hint: game.i18n.localize("SWFFG.Crew.Settings.Hint"),
+    icon: "fas fa-file-import",
+    type: CrewSettings,
+    restricted: true,
+  });
+
+  game.settings.register("starwarsffg", "arrayCrewRoles", {
+    module: "starwarsffg",
+    name: "arrayCrewRoles",
+    scope: "world",
+    default: defaultArrayCrewRoles,
+    config: false,
+    type: Object,
+  });
+}
```

Comment:
Cleaner to break this into its own helper class file somewhere, but fine for now... swffg-main is already mostly bloat.

Reply:
Ack